packages/core/src/polyfills.ts

```diff
@@ -0,0 +1,17 @@
+export function withResolvers() {
```

Comment:
This is here so that the `withResolvers` functionality is supported in older browsers, do I have that right?